hs_core/views/resource_rest_api.py

```diff
@@ -156,6 +156,8 @@ def get_queryset(self):
         filter_parms['user'] = (self.request.user if self.request.user.is_authenticated() else None)
         if len(filter_parms['type']) == 0:
             filter_parms['type'] = None
+        else:
+            filter_parms['type'] = list(filter_parms['type'])
```

Comment:
This is required because hydroshare.get_resource_list can't handle multiple-value filter parameters that are sets (which the serializer returns for this field), but instead requires that lists be used.